docs/usage_guide.rst

```diff
@@ -1370,6 +1370,19 @@ Before v1.6.0 rotki was saving data in ``$USER/.rotkehlchen``. From v1.6.0 that
 
 A very good idea about the rotki data directory would be to be making frequent backups of it as it contains all of the data of all of your rotki accounts and cache data for historical price queries.
 
+Manually accessing the database
+*******************************
+
+Is it possible that to detect possible problems with data or modify the state of the database the user needs to manually access it. To do so we recommend the usage of `DB Browser for SQLite <https://sqlitebrowser.org/dl/>`_.
+DB Browser for SQLite is an open source software that provides an accessible UI and brings two programs, one with the normal browser and another with the functionalities to access encrypted databases. You can download it
+for Windows, MacOS and Linux at the previous link.
+
+Once installed you have to start the program with the sqlcipher functionality and after that you need to select the rotki database, this database can be found in the rotki data directory. Once you open it, the program will
```

Comment:
1. Give an internal link to the section where the information for the data directory is: https://rotki.readthedocs.io/en/latest/usage_guide.html#rotki-data-directory
2. Syntax
```suggestion
Once installed you have to start the program with the sqlcipher functionality and after that you need to select the rotki database. This database can be found in the rotki data directory. Once you open it, the program will
```